docs/scenarios_authoring/HandlingSchemas.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 11
+---`
+
+# Handling Schemas
+
+## Overview
+
+Nussknacker supports two types of schema:
+* [JSON Schema](https://json-schema.org/) in version: `Draft 7`
+* [Avro Schema](https://avro.apache.org/) in version: `1.11.0`
+
+Avro Schema and JSON Schema are available on [Streaming Source/Sink](/docs/scenarios_authoring/DataSourcesAndSinks.md)
+in contrast to [RequestResponse Source/Sink](/docs/scenarios_authoring/RRDataSourcesAndSinks.md) where available is just only one - JSON Schema.
+
+Nussknacker converts `source schema` to its own static `Typing` information to create hinting on the FE Designer,
+whereas `sink schema` is used to validate output data.
+
+## Source conversion
+Conversion at source to specific type means that behind the scene Nussknacker converts `primitive type` to `logical type`
```

Comment:
I would simplify this by removing references to methods - just write about type conversion. 

Terms _primitive type_ and _logical type_ do not help to understand the subject of conversion. Now you need to explain these terms. I would not explore this subject (primitive, logical types) at all. Just type conversion. May be you should write _silent conversion_ because this is what is happening here - without explicit instruction in the (expression) code, the type gets converted. 
Finally, this stuff on conversion should be written in the context of schema validation. So what is the bottom line of these conversions? Why are they mentioned here?